Dockerfile

```
@@ -1,4 +1,6 @@
-FROM gcr.io/kaniko-project/executor:debug-v0.19.0
+# v1.6.0-debug
+ARG EXECUTOR_DIGEST=fcccd2ab9f3892e33fc7f2e950c8e4fc665e7a4c66f6a9d70b300d7a2103592f
+FROM gcr.io/kaniko-project/executor@sha256:$EXECUTOR_DIGEST
```

usually we put the `sha256:` in the digest variable
```suggestion
ARG EXECUTOR_DIGEST=sha256:fcccd2ab9f3892e33fc7f2e950c8e4fc665e7a4c66f6a9d70b300d7a2103592f
FROM gcr.io/kaniko-project/executor@$EXECUTOR_DIGEST
```